lib/services/lookup.js

```diff
@@ -482,6 +482,26 @@ function lookupServiceFactory(
     };
 
     /**
+     * Returns a list of object containing IP and Mac Addresses for the given Node ID.
+     * @param  {String} id Node ID
+     * @return {Promise.<Array>} A promise fulfilled with an array of
+     * objects containing IP Addresses and Mac Addresses associated with the Node.
+     */
+    LookupService.prototype.findIpMacAddresses = function (nodeId) {
+        var self = this;
+
+        return self.validateArpCache()
+        .then(function() {
+            return waterline.lookups.find({node: nodeId}).then(function (records) {
+                return _.reduce(records, function (ipMacAddresses, record) {
+		     ipMacAddresses.push(_.pick(record, ['ipAddress', 'macAddress']));
```

Comment:
indent problem. I strongly suggest you checking your editor configuration, since there are usually a lot indent problem in your recent PRs.